torchvision/models/quantization/googlenet.py

```diff
@@ -74,6 +74,8 @@ def forward(self, x: Tensor) -> Tensor:
 class QuantizableGoogLeNet(GoogLeNet):
     # TODO https://github.com/pytorch/vision/pull/4232#pullrequestreview-730461659
     def __init__(self, *args: Any, **kwargs: Any) -> None:
+        if "init_weights" not in kwargs:
+            kwargs["init_weights"] = True
```

Comment:
I'm not sure why in order to fix the warning on unit-tests we modify the models themselves. My expectations without having all the details in my mind, is that the tests are the ones that need to be updated. What am I missing?

Reply:
At first I was thinking that QuantizableGoogLeNet does not properly set up its base class `GoogLeNet`. Thinking more about that, I think you are right, we have to try again to fix that from tests side

Reply:
Ok, let me check and send PR again.